Gemfile

```diff
@@ -21,13 +21,18 @@ gem 'jquery-rails'
 
 # Use datatables for rendering and searching songs
 gem 'jquery-datatables-rails', '~> 3.3.0'
+gem 'will_paginate'
 
+gem 'yomu'
+gem 'tesseract-ocr'
```

Comment:
Lol yup! The PDFs are not text based, theyre just scans of song sheets...so
pretty much images. PDF reader libs couldnt grab the text so I had to use
OCR.

On Sunday, August 14, 2016, Patrick <notifications@github.com> wrote:

> In Gemfile
> <https://github.com/GracepointMinistries/GraceTunes/pull/10#discussion_r74720093>
> :
>
> >
> > +gem 'yomu'
> > +gem 'tesseract-ocr'
>
> WOAH you had to use OCR to parse the PDF files?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GracepointMinistries/GraceTunes/pull/10/files/a3c4279f02dd80f6b8ecfd786c7ce88182b5e5be#r74720093>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFXKhI-OR73xt_wgwGat8LpI6Oblnkxfks5qf-39gaJpZM4JkAcm>
> .
>